src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -149,8 +150,11 @@ let on_transition t ~sender ~root_sync_ledger ~genesis_constants
         with
         | Ok (`Root root, `Best_tip best_tip) ->
             if done_syncing_root root_sync_ledger then return `Ignored
-            else
+            else (
+              t.num_of_root_snarked_ledger_retargeted <-
+                t.num_of_root_snarked_ledger_retargeted + 1 ;
```

Comment:
Should we do this only in the `New` case in `start_sync_job_with_peer`?

Reply:
Yes. That's better.